LICENSE

```diff
@@ -36,6 +36,7 @@ The following files and directories are exceptions:
 * vendor/assets/javascripts/jquery.jeditable.js is MIT licensed
 * vendor/assets/javascripts/jquery.jnotify.js is ASL 2.0 licensed
 * vendor/assets/javascripts/jstz.js is MIT licensed
+* vendor/assets/javascripts/ipaddr.js is MIT licensed
```

Comment:
Sure, I could do that. To me it looks like a little overkill, though. It's just a single js file. Why is a gem asset better?